app/assets/styles/pool/create.scss

```diff
@@ -171,3 +171,90 @@ bl-vm-size-picker {
         }
     }
 }
+
+bl-autoscale-formula-picker {
```

Comment:
I just merged a new way to handle styles. So you can use angular2 `styleUrls` check the `tags.component.ts` for a sample or the `components.md` doc. This will make styles cleaner